src/main/java/org/frc5687/infiniterecharge/robot/OI.java

```diff
@@ -91,8 +91,8 @@ public void initializeButtons(Shifter shifter, DriveTrain driveTrain, Turret tur
         _operatorRightBumper.toggleWhenPressed(new ShootSpeedSetpoint(shooter, this, 1.0));
         _operatorRightTrigger.whenHeld(new Shoot(shooter, indexer, turret, this));
 
-        _driverLeftBumper.whenPressed(new RaiseIntake(intake));
-        _driverRightBumper.whenPressed(new LowerIntake(intake));
+//        _driverLeftBumper.whenPressed(new RaiseIntake(intake));
+//        _driverRightBumper.whenPressed(new LowerIntake(intake));
```

Comment:
Did you mean to leave these commented out?

Reply:
These will be changed to shift. I'm pretty sure they are changed in #50